lib/widgets/movie_detail_image.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:flutter/material.dart';
+import '../constants/measures_constants.dart';
+
+class MovieDetailImage extends StatelessWidget {
+  final Widget image;
+
+  const MovieDetailImage({
+    Key key,
+    this.image,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: [
+        Container(
+          height: MeasuresConstants.movieDetailContainerHeight,
+          child: Center(
+            child: image,
```

Comment:
Here you can add the Hero widget that you create in home_body_content

Reply:
Added